### PR TITLE
rename additional product terms and alphabetize them

### PR DESCRIPTION
**Pull request by vollmera**
refresh of the additional product terms, including
- rename GitHub Additional Product Terms to GitHub Terms for Additional Products and Features
- put products/features in alphabetical order
- remove products/features with terms/documentation covered elsewhere (Marketplace, Third Party Integrations, GitLFS, Spectrum)
- add npm

### CODE REVIEW

File: Policies/github-additional-product-terms.md [hunk 1]
```diff
@@ -75,53 +34,76 @@ Actions and any elements of the Actions service may not be used in violation of
...
+#### a. Bandwidth and Usage Limits
+GitHub Pages are subject to some specific bandwidth and usage limits, and may not be appropriate for some high-bandwidth uses. Please see our [GitHub Pages guidelines](/github/working-with-github-pages/about-github-pages) for more information. 
 
-#### b. Third-party components
+#### b. Prohibited Uses
+Prohibited uses of GitHub Pages include
+- Content or activity that is illegal or otherwise prohibited by our [Terms of Service](/github/site-policy/github-terms-of-service), [Acceptable Use Policies](/github/site-policy/github-acceptable-use-policies) or [Community Guidelines](/github/site-policy/github-community-guidelines)
+- Violent or threatening content or activity
+- Excessive automated bulk activity (for example, spamming)
+- Activity that compromises GitHub users or GitHub services
```
Comment (sickcodes) [inline Policies/github-additional-product-terms.md:96]: The word, "compromises," is not quite clear in this sentence and should be refined.

File: Policies/github-additional-product-terms.md [hunk 2]
```diff
@@ -1,72 +1,31 @@
...
-  - legal
+  - Policy
+  - Legal
 ---
 
-Version Effective Date: February 22, 2021
+Version Effective Date: May 26, 2021
 
-When you create an Account, you're given access to lots of different features and products that are all a part of the Service. Because many of these features and products offer different functionality, they may require additional terms and conditions specific to that feature or product. Below, we've listed those features and products, along with the corresponding additional terms that apply to your use of them.
+When you use GitHub, you may be given access to lots of additional products and features ("Additional Products and Features"). Because many of the Additional Products and Features offer different functionality, specific terms for that product or feature may apply in addition to your main agreement with us—the GitHub Terms of Service, GitHub Corporate Terms of Service, GitHub General Terms, or Microsoft volume licensing agreement (each, the "Agreement"). Below, we've listed those products and features, along with the corresponding additional terms that apply to your use of them.
```
Comment (sickcodes) [inline Policies/github-additional-product-terms.md:14]: What is the, "Microsoft volume licensing agreement"?

Are all users required to agree to the, "Microsoft volume licensing agreement"?

If so, could you please provide a link to that agreement?

**Review by sickcodes — CHANGES_REQUESTED** [review]

File: Policies/github-additional-product-terms.md (hunk 1 above)
Comment (vollmera) [reply Policies/github-additional-product-terms.md:96]: 👍 @sickcodes - We'll 🤔 about how to make this clearer. I opened https://github.com/github/site-policy/issues/408

File: Policies/github-additional-product-terms.md (hunk 2 above)
Comment (vollmera) [reply Policies/github-additional-product-terms.md:14]: @sickcodes the Microsoft volume licensing agreement is for Enterprise customers. 
>Are all users required to agree to the, "Microsoft volume licensing agreement"?

No - if you have a GitHub account, you've agreed to one of these (there's an "or" there)
>your main agreement with us—the GitHub Terms of Service, GitHub Corporate Terms of Service, GitHub General Terms, or Microsoft volume licensing agreement (each, the "Agreement")

so we are saying you this doc provides terms that are additional to whichever one of those is your main agreement.

**Review by lee-dohm — APPROVED** [review]
🚢

**Comment by vollmera** [comment]
Thanks for the feedback! The comment period has ended and the changes are now live 🎉

**Review by artursavin86 — COMMENTED** [review]
github-additional-product-terms.md

**Review by ig33 — COMMENTED** [review]
Support

**Comment by Kixwag-2vappy-hiMxam** [comment]
- [ ] github additional product terms and GitHub terms update main

**Comment by artursavin86** [comment]
#406

**Comment by chole22** [comment]
I'm

**Review by chadhall2152 — COMMENTED** [review]
#446

**Comment by ReenaSuoliman** [comment]
Thank you

**Review by chadhall2152 — COMMENTED** [review]
Policy

**Pull request merged**